pymc4/model/base.py

```diff
@@ -67,6 +68,41 @@ def get_mode(state, rv, *args, **kwargs):
             returns = self.session.run(list(values_collector.result.values()))
         return dict(zip(values_collector.result.keys(), returns))
 
+    def log_prob_fn(self, x,  *args, **kwargs):
```

Comment:
hmm, I'm not sure this will work. ancestors of RV depend on the RV, here you do not replace RV with `value=kwargs.get(i)`